demos/demo.html

```diff
@@ -1,3 +1,4 @@
+<link rel="stylesheet" href="/public/main.css">
```

Comment:
**Question**: I didn't like needing to do this. I was expecting a manifest.json file in the `public` folder. I think I would need to do this on every single html file.